src/app/App.css

```diff
@@ -1,5 +1,24 @@
 .App {
```

Comment:
Will this change effect the style of the whole app? Would it make sense to have a separate style for the timeline?

Reply:
@hollyegrayson, yes you're absolutely right, it will affect the entire app.  I think we will need to adjust the styles to be more agreeable over the next week.  But we haven't really given it much thought to this point.  Perhaps we can figure more or less what we want the overall experience to look like.  But I think for now, it's ok to change this css.

Reply:
That sounds like a good plan to me.